src/components/CategoryInput/ListItem.jsx

```diff
@@ -0,0 +1,11 @@
+import React, {Component} from "react";
+
+class ListItem extends Component {
```

Comment:
I would argue that this component isn't that necessary if it's just returning a single element. Only abstract things if it makes organizational sense.

Reply:
not too sure how to use map without using props :(